### PR TITLE
IdCatchupEntryProcessor should use IDS table

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public Void process( Entry<Integer, Range> entry ) {
 
     public PreparedStatement prepareExistQuery( Connection connection ) throws SQLException {
         return connection.prepareStatement(
-                "SELECT count(*) from " + PostgresTable.ENTITY_KEY_IDS.getName() +
+                "SELECT count(*) from " + PostgresTable.IDS.getName() +
                         " WHERE " + PostgresColumn.ID_VALUE.getName() + " = ?" );
```
Comment (you-leee) [inline src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java]: Maybe add also filter for partition?

Comment (katherinebernstein) [reply src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java]: 👍👍👍

Comment (katherinebernstein) [reply src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java]: Although actually we'd need to look up the entity set ids in order to load the partitions, which would require doing this same query anyway

Comment (you-leee) [reply src/main/java/com/openlattice/ids/IdCatchupEntryProcessor.java]: Yeah, right!

**Comment by katherinebernstein** [comment]
Yeah, I didn't touch it since whoever wrote it left that comment but kept the checkNotNull anyway and I'm not sure why...

**Pull request merged**